k8s/upgrade/src/bin/upgrade-job/helm/chart.rs

```diff
@@ -82,10 +82,10 @@ pub(crate) struct CoreValues {
     /// This contains Kubernetes CSI sidecar container image details.
     csi: Csi,
     /// The contains the values for the jaegertracing/jaeger-operator chart.
-    #[serde(rename(deserialize = "jaeger-operator"))]
+    #[serde(default, rename(deserialize = "jaeger-operator"))]
     jaeger_operator: JaegerOperator,
     /// This contains loki-stack details.
-    #[serde(rename(deserialize = "loki-stack"))]
+    #[serde(default, rename(deserialize = "loki-stack"))]
```

Comment:
Can we set default on the struct itself, does that work?

Reply:
Yes!

Reply:
I'm going avoid slapping the default on the whole values set.
But I am going to add it to all of the dependent charts' values, as those cause all of the trouble helm 3.13 onwards.